gcloud/storage/_helpers.py

```diff
@@ -52,12 +52,20 @@ def __init__(self, name=None):
         self._properties = {}
         self._changes = set()
 
-    def reload(self):
-        """Reload properties from Cloud Storage."""
+    def reload(self, connection=None):
+        """Reload properties from Cloud Storage.
+
+        :type connection: :class:`gcloud.storage.connection.Connection`
+        :param connection: An explicit connection to use for the API request.
+                           If not passed, use the connection assigned to
+                           the object in its constructor.
+        """
+        if connection is None:
+            connection = self.connection
```

Comment:
That just makes us change this line twice instead of once. I prefer `_require_connection` once (and for all).